spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -3,7 +3,8 @@ class TestAppGenerator < Rails::Generators::Base
   source_root File.expand_path("../../../../spec/test_app_templates", __FILE__)
 
   def install_blacklight
-    generate 'blacklight:install --devise'
+    # we can skip solr because the activefedora installer will also do it (in the hydra:head generator)
```

Comment:
I didn't dig into the exact AF versions... Is this true?

Reply:
I checked and this is true.